src/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
We need an automatic way to detect package vulnerabilities and management (FOSSA)

Reply:
Opened tracking issue #5